From 28df4186c6cf782496ccfc74320b18435dacc2e5 Mon Sep 17 00:00:00 2001
From: Arnaud Patard <arnaud.patard@rtp-net.org>
Date: Tue, 19 Jul 2011 09:44:41 +0200
Subject: [PATCH 1/2] Fix get_db_handle

Commit 72f987c0a03e58358ea616628514efa0113f0332 moved Client class from
Sbuild::DB to Buildd:: and now, when uploading, there are errors in the
log:
Can't locate object method "new" via package "Buildd::Client" (perhaps you
forgot to load "Buildd::Client"?) at /usr/share/perl5/Buildd/Base.pm line 120 (#1)

Adding a "use Buildd:Client;" cures the problem for me.

Signed-off-by: Arnaud Patard <arnaud.patard@rtp-net.org>
---
 lib/Buildd/Base.pm |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/lib/Buildd/Base.pm b/lib/Buildd/Base.pm
index 2ed134c..91d831f 100644
--- a/lib/Buildd/Base.pm
+++ b/lib/Buildd/Base.pm
@@ -25,6 +25,7 @@ use warnings;
 
 use IO::File;
 use Buildd qw(lock_file unlock_file);
+use Buildd::Client qw();
 
 use Sbuild::Base;
 use Sbuild qw($devnull);
-- 
1.7.4.1

